src/main/java/com/assignment/application/controller/ProjectController.java

```diff
@@ -13,22 +16,68 @@
 public class ProjectController {
 
     @Autowired
-    ProjectServiceI projectServiceI;
+    private ProjectServiceI projectServiceI;
 
-    @DeleteMapping(value="/{project_id}" )
+    @Autowired
+    private VerifyUser verifyUser;
```

Comment:
VerifyUser, seems to me like a value object, not a service. This is sort of a authentication helper.